src/cosmic_ray/cli.py

```diff
@@ -84,7 +84,10 @@ def init(config_file, session_file):
     executed with other commands.
     """
     cfg = load_config(config_file)
-
+    try:
```

Comment:
I think this check is better handled in `load_config()` itself. I.e. `load_config()` should always return a config with a `operators_config` field, even if the value is `None`. That way we don't have to duplicate this logic in other (potential) places where `load_config()` is called.